axelor-human-resource/src/main/resources/views/Timesheet.xml

```diff
@@ -4,8 +4,8 @@
 
 	<grid name="timesheet-grid" title="Timesheet" model="com.axelor.apps.hr.db.Timesheet" canNew="false" groupBy="user" orderBy="user,-fromDate">
 		<menubar>
-			<menu title="Tools" icon="fa-wrench" showTitle="true">
-				<item action="action-timesheet-method-timesheet-line-historic" title="See timesheet lines"/>
+			<menu name="timeSheetToolsMenu" title="Tools" icon="fa-wrench" showTitle="true">
+				<item name="seeTimeSheetLinesItem" action="action-timesheet-method-timesheet-line-historic" title="See timesheet lines"/>
```

Comment:
'Timesheet' is a complete word, so don't write 'TimeSheet' keep it 'Timesheet'. 

Also, check the same whenever it is needed.